program/src/state.rs

```diff
@@ -1622,6 +1622,9 @@ impl MangoAccount {
         pa.change_base_position(perp_market, base_change);
         let quote = I80F48::from_num(perp_market.quote_lot_size.checked_mul(quote_change).unwrap());
         let fees = quote.abs() * fill.maker_fee;
+        if !fill.market_fees_applied {
```

Comment:
so iiuc, with this change you are accounting for those events which were about to be processed before deploying last night, or? havent all these events now already passed? are any left to enter this if branch?

Reply:
update: of course, those which are still open orders, got it